app/view/sdl/SubtleAlertPopUp.js

```diff
@@ -253,15 +237,38 @@ SDL.SubtleAlertPopUp = Em.ContainerView.create(
                 }
             }
             this.set('active', true);
-            this.set('timeout', message.duration ? message.duration : 10000);
-            this.set('endTime', Date.now() + this.timeout);
-            clearTimeout(this.timer);
-            this.timer = setTimeout(
-                function () {
-                    self.deactivate(self.reason, self.message);
-                }, this.timeout
-            );
-            window.addEventListener('click', this.onClick);
+            this.setTimerUI(message.duration ? message.duration : this.defaultTimeout);
+        },
+
+        /*
+        * function setTimerTTS. Sets the active timer of the view for TTS RPC
+        */
+        setTimerTTS: function(time){
+        var self = SDL.SubtleAlertPopUp;
+        self.set('ttsTimeout', time);
+        clearTimeout(self.ttsTimer);
+        self.ttsTimer = setTimeout(
+          function() {
+            clearTimeout(self.ttsTimer);
+          }, self.ttsTimeout
+        );
```

Comment:
@VadymLuchko Add indentation.

Reply:
fixed in 9ad1a683d7504e8708d877a776583bfcdf564ce2